elvis.config

```diff
@@ -3,14 +3,28 @@
   elvis,
   [
    {config,
-    [#{dirs => ["apps/*/src", "apps/*/test", "test/ejabberd_tests/tests"],
+    [#{dirs => ["apps/*/src"],
        filter => "*.erl",
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
 		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
 		]
-      }
+      },
+     #{dirs => ["apps/*/test", "test/ejabberd_tests/tests"],
+       filter => "*.erl",
+       ruleset => erl_files,
+       rules => [{elvis_style, line_length, #{limit => 100,
+                                              skip_comments => false}},
+                 {elvis_style, dont_repeat_yourself, disable},
+                 {elvis_style, invalid_dynamic_call, disable},
+                 {elvis_style, no_debug_call, disable},
+                 {elvis_style, god_modules, disable},
```

Comment:
I'd rather see Elvis unhappy for this. You don't have to fix it in this PR, you already put a big effort in improving the code according to Elvis's suggestions.